Configured the rubocop

EvgeniiMaiorov opened the pull request:
Configure the rubocop

Commit:
Configured the rubocop
1. Added gems 'rubocop', 'rubocop-rails', 'rubocop-rspec', 'rubocop-performance'
2. Created and configured the configuration file .rubocop.yml

anevsevra commented on .rubocop.yml:
```diff
@@ -0,0 +1,19 @@
+Rails:
+  Enabled: true
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
```
What is the point disabling this rule? It is good practice to have string literals frozen.

Commit:
Enable Style/FrozenStringLiteralComment rule

1. Delete rule Style/FrozenStringLiteralComment. Delete rule from .rubocop.yml
2. Fix rubocop warnings
